README.md

```diff
@@ -162,6 +162,23 @@ Per compilare la libreria e generare i file nella cartella `dist`, è sufficient
 $ yarn build
 ```
 
+## Supporto browsers
+
+Come da indicazioni riportate nelle Linee Guida di Design per i servizi web della Pubblica Amministrazione, sezione [6.3.1.2.1. Supporto browser](https://docs.italia.it/italia/designers-italia/design-linee-guida-docs/it/2020.1/doc/user-interface/lo-sviluppo-di-un-interfaccia-e-i-web-kit.html#strumenti), di seguito è indicata la lista dei browser supportata dal Design Kit, disponibile nel formato `browserslist`:
+
+```json
+"browserslist": [
+    "defaults",
+    "ie 11",
+    "not ie_mob 11",
+    "not op_mini all",
+    "edge >= 13",
+    "safari 11"
+]
+```
+
+La lista è anche disponibile nel `package.json`. Purtroppo, non è ancora possibile estendere la lista di questo pacchetto per [problematiche di sicurezza](https://github.com/browserslist/browserslist#shareable-configs).
```

Comment:
Questo cosa implica o in cosa ci limita? È necessario esplicitarlo nel readme?

Reply:
Al momento è nel readme perchè non è possibile estendere la lista dal `design-react-kit` a meno di disabilitare i controlli di sicurezza (opzione possibile ma non mi sentirei di consigliarla).
Questa PR è una via di mezzo che mette a disposizione un qualcosa da copiare e incollare (e avendolo nel `package.json` il dev potrebbe anche farsi uno script suo).

Se fosse un'opzione plausibile, magari in condivisione con BI, si potrebbe pensare ad un pacchetto `browserlist` estendibile ufficialmente, che sia `design-react-kit`, che `bootstrap-italia` ed eventualmente i progetti PA che li usano potrebbero usare.

Reply:
Ok, sì sarebbe carino. Al momento la gestion su [BI è infatti un po' più minimale](https://github.com/italia/bootstrap-italia/blob/master/package.json#L38). Cmq tutto ok, si può mergiare.